includes/Core/Util/Debug_Data.php

```diff
@@ -88,6 +97,7 @@ public function __construct(
 		$this->options        = $options ?: new Options( $this->context );
 		$this->user_options   = $user_options ?: new User_Options( $this->context );
 		$this->authentication = $authentication ?: new Authentication( $this->context, $this->options, $this->user_options );
+		$this->credentials    = $this->authentication->credentials();
```

Comment:
This isn't necessary, and we don't assign instance properties directly from another class anywhere else in a constructor. Please remove this line (and the added property above).

Reply:
Fixed